src/App.tsx

```diff
@@ -1,27 +1,106 @@
-import { render } from '@testing-library/react';
-import React from 'react';
-import './App.css';
-import Contacts from './Contacts';
-import Header from './Header';
-import Sidebar from './Sidebar';
-import Feed from './Feed';
+import React, { useEffect, useState } from "react";
+import "./App.css";
+import Contacts from "./Contacts";
+import Header from "./Header";
+import Sidebar from "./Sidebar";
+import Feed from "./Feed";
+import CreatePostWindow from "./CreatePostWindow";
 
 function App() {
+  let username = "Meir";
+
+
+  
+  const [postData, setPostData] = useState([""]);
+  const addPost = (input: string) => {
+    setPostData([input, ...postData]);
+  };
+
+  const [isNewPostWindowAlive, setWindowState] = useState(false);
```

Comment:
the state and setState function must be in this pattern: stateName, setStateName.
so change it to [isNewPostWindowAlive, setIsNewPostWindowAlive].

Reply:
okay, I will change it, it's just I felt out of names, due to the need to name the "handlers", too


(do you understand what I mean by saying 'handlers'?
 I refer to functions of this type ("**setLocalTemp**'):
```
  const [localTempNewPost, setLocalTempNewPost] = useState("");
  const setLocalTemp = (input: string) => {
    setLocalTempNewPost(input);
  }; 
```
perhaps it's called callback? I don't know
)

Reply:
If you don't do anything in this callback besides setState you do not need it. You can use directly the setState